requirements/prefect.txt

```diff
@@ -0,0 +1,4 @@
+prefect==3.0.2
```

Comment:
Alternatively, we can start with a `prefect.in` and generated `prefect.txt` right away here.

We could also make that a separate change, but I do not see much momentum for any type of refactoring.